packages/flutter/test/material/search_test.dart

```diff
@@ -91,6 +91,23 @@ void main() {
     expect(find.text('Suggestions'), findsOneWidget);
   });
 
+  testWidgets('Hint text color overridden', (WidgetTester tester) async {
+    final _TestSearchDelegate delegate = _TestSearchDelegate();
+
+    await tester.pumpWidget(TestHomePage(
+      delegate: delegate,
+    ));
+    await tester.tap(find.byTooltip('Search'));
+    await tester.pumpAndSettle();
+    expect(
+        tester
```

Comment:
The format of this is a little tough to follow. I suggest something like this:

```
final TextField textField = tester.widget<TextField>(find.byType(TextField));
final Color hintColor = textField.decoration.hintStyle.color;
expect(hintColor, delegate.hintTextColor);
```

Reply:
Done.